challenges/__tests__/tree.test.js

```diff
@@ -47,3 +47,5 @@ describe('testing methods on both trees', () => {
     expect(tree.maxValue()).toEqual(4);
   });
 });
+
+///////////////////////////////////////////////////////////////////////////////////
```

Comment:
I am having an issue when I try and run your tests here is part of it and where the issue is coming from:

```
    SyntaxError: /Users/adrianhuebner/CodeFellows/40-401/students/seid/data-structures-and-algorithms/challenges/tree/tree.js: Unexpected token (112:6)

      110 | //////////////////////////////////////////////////////////////////////////////
      111 | 
    > 112 | class BinaryTree{
          |       ^
      113 |   constructor(){
      114 |     this.root = null;
      115 |   }
```